evadb/third_party/databases/postgres/postgres_handler.py

```diff
@@ -0,0 +1,112 @@
+# coding=utf-8
```

Comment:
Since this is in the same folder as types.py and interface.py, shall we move this to a separate dir

evadb/third_party/databases/handlers/postgres_handler/.....

This would also make it cleaner while adding handlers for other backends

Reply:
The file structure is inspired by https://github.com/mindsdb/mindsdb/tree/staging/mindsdb/integrations/handlers